packages/main/src/DateRangePicker.js

```diff
@@ -3,6 +3,7 @@ import Integer from "@ui5/webcomponents-base/dist/types/Integer.js";
 import ValueState from "@ui5/webcomponents-base/dist/types/ValueState.js";
 import CalendarDate from "@ui5/webcomponents-localization/dist/dates/CalendarDate.js";
 import DateRangePickerTemplate from "./generated/templates/DateRangePickerTemplate.lit.js";
+import RenderScheduler from "../../base/src/RenderScheduler.js";
```

Comment:
This breaks several builds, please use the "@ui5/webcomponents-base/dist/RenderScheduler.js" form

Reply:
I will for the future. Thanks.